client/run-cypress-tests.sh

```diff
@@ -3,7 +3,7 @@
 export ARLO_AUDITADMIN_AUTH0_BASE_URL=https://votingworks-noauth.herokuapp.com
 export ARLO_JURISDICTIONADMIN_AUTH0_BASE_URL=https://votingworks-noauth.herokuapp.com
 
-trap 'kill 0' SIGINT SIGHUP EXIT
+trap 'exit 0' SIGINT SIGHUP EXIT
```

Comment:
im pretty sure the `kill 0` was there to kill all the child processes as well. have you tried running this locally? does it leave zombie processes?

Reply:
that's weird, because i dont think it did that before, did it? i.e. cypress tests have been passing on CI